external/storm-cassandra/src/main/java/org/apache/storm/cassandra/trident/state/TridentResultSetValuesMapper.java

```diff
@@ -44,6 +44,8 @@ public TridentResultSetValuesMapper(Fields outputDeclaredFields) {
     @Override
     public List<List<Values>> map(Session session, Statement statement, ITuple tuple) {
         List<List<Values>> list = new ArrayList<>();
+        List<Values> innerList = new LinkedList<>();
```

Comment:
Could you explain that how different the changeset is? I don't get it and for me it looks like same.